src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/OutputMode.java

```diff
@@ -10,7 +10,8 @@
     /** produces calls at variant sites and confident reference sites */
     EMIT_ALL_CONFIDENT_SITES,
 
-    /** produces calls at any callable site regardless of confidence; this argument is intended only for point
-     * mutations (SNPs); it will not produce a comprehensive set of indels. */
-    EMIT_ALL_SITES
+    /** produces calls at any callable site regardless of confidence. This does not necessarily output calls for all
```

Comment:
`produces calls at any callable site regardless of confidence` -> `Produces calls at any region over the activity threshold regardless of confidence`

Reply:
done